pyltc/struct.py

```diff
@@ -112,12 +112,18 @@ class Filter(object):
     """Represents a filter node in the LTC chain structure."""
 
     _handle_counter = 1
+    _prio_counter = 1
 
     def __init__(self, name, parent, cond, flownode, **kw):
         # TODO: implement proper data handling
         self._handle = self.__class__._handle_counter
         self.__class__._handle_counter += 1
 
+    @classmethod
+    def prio_counter(cls):
+        cls._prio_counter += 1
+        return cls._prio_counter - 1
```

Comment:
I would think there's a more elegant way to accomplish returning the previous count, the code reads a bit awkwardly here.